x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.ts

```diff
@@ -77,57 +76,227 @@ const getEnabledRulesByBenchmark = (rules: SavedObjectsFindResponse<CspRule>['sa
     return benchmarks;
   }, {});
 
+/* @internal */
 export const createRulesConfig = (
-  cspRules: SavedObjectsFindResponse<CspRule>
+  cspRules: Array<SavedObject<PackagePolicyRuleUpdatePayload>>
 ): CspRulesConfiguration => ({
   runtime_cfg: {
-    activated_rules: getEnabledRulesByBenchmark(cspRules.saved_objects),
+    activated_rules: getEnabledRulesByBenchmark(cspRules),
   },
 });
 
-export const convertRulesConfigToYaml = (config: CspRulesConfiguration): string => {
-  return yaml.safeDump(config);
-};
-
-export const setVarToPackagePolicy = (
+/* @internal */
```

Comment:
i used `@internal` to annotate exported functions in order to indicate these are exported for testing, as opposed to exported functions without it.